src/main/resources/templates/employee-view.html

```diff
@@ -28,7 +28,7 @@ <h1>Enter Weekly Payroll</h1>
         <input type="text" name="hours" th:field="*{hoursWorked}"/>
         <label>Amount Paid</label>
         <input type="text" name="hours" th:field="*{amountPaid}"/>
-        <button name="submit" type="submit" value="0" class="btn btn-primary">Submit</button>
+        <button name="submit" type="submit" value="0" required= "required"/> class="btn btn-primary">Submit</button>
     </form>
```

Comment:
I put a required validation in there. So employees will have to put in there information before moving forward.

Reply:
I think this is a fantastic change.